recipes/rdock/build.sh

```diff
@@ -1,30 +1,18 @@
-#!/bin/bash
+cd build
+chmod +x p4utils.pl
 
-if [ $PY3K -eq 1 ]; then
-    2to3 -w -n build/test/RBT_HOME/check_test.py
-fi
-export C_INCLUDE_PATH=${PREFIX}/include
-export CPP_INCLUDE_PATH=${PREFIX}/include
-export CPLUS_INCLUDE_PATH=${PREFIX}/include
-export CXX_INCLUDE_PATH=${PREFIX}/include
-export LIBRARY_PATH=${PREFIX}/lib
+# Needs -std=c++14 because of
+#   error: ISO C++17 does not allow dynamic exception specifications
+export CXXFLAGS="${CXXFLAGS} -std=c++14"
```

Comment:
This is probably not yet needed since we still use GCC 10 (`-std=c++17` is the new default for GCC 11+). I used GCC 12 locally and we'll update to that one some time in the next months, I assume.